Pods/Alamofire/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2014-2017 Alamofire Software Foundation (http://alamofire.org/)
```

Comment:
@sebikaplun i know that sometimes committing the pods is good practice, but not sure we need it in this case. wdyt?

Reply:
@ShayDavidson IMO if this project will be for an interview, the fastest way to start working on it is with all the files on it. Download the project, press run, and everything should work.

This will help the candidate to avoid installing cocoapods, and the pods

Reply:
you're right but my intent is to actually see if the candidate is familiar with cocoapods ;)